omnibus/config/software/datadog-agent.rb

```diff
@@ -117,6 +117,14 @@
     command "invoke -e process-agent.build --python-runtimes #{py_runtimes_arg} --major-version #{major_version_arg} --arch #{platform}", :env => env
 
     copy 'bin/process-agent/process-agent.exe', "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
+
+    unless windows_arch_i386?
+      if ENV['WINDOWS_DDFILTER_DRIVER'] and not ENV['WINDOWS_DDFILTER_DRIVER'].empty?
+        ## don't bother with system probe build on x86.
+        command "invoke -e system-probe.build --windows"
+        copy 'bin/system-probe/system-probe.exe', "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
+      end
+    end
```

Comment:
I'm not too familiar with how we approach impossible build scenarios in the omnibus build, but should we have a louder failure if you try to build the windows sysprobe on i386 ? This looks like it will silently leave the system probe out.

Reply:
That's the point.  If we're building an i386 version, we shouldn't even build the sysprobe.  We're not going to include it (there's no x86 driver).  It's not a failure, we just ignore the sysprobe options on i386